config/default.yml

```diff
@@ -53,6 +53,13 @@ Sorbet/FalseSigil:
   - db/**/*.rb
   - script/**/*
 
+Sorbet/ForbidExtendTSigInShims:
+  Description: 'Forbid the use of `extend T::Sig` in RBI shims'
+  Enabled: false
```

Comment:
We can safely enable this as well. If anyone is doing it, they are doing it wrong.